lua/hurl/main.lua

```
@@ -277,7 +303,7 @@ function M.setup()
       for name, value in pairs(persisted_vars) do
         if not env_vars[name] then
           table.insert(lines, name .. ' = ' .. value)
-        end
+        endbefore each request (
```

Luacheck (1.2.0)

[error] expected '=' near 'each'

(E011)